internal/server/models/settings.go

```diff
@@ -1,39 +1,9 @@
 package models
 
-import (
-	"github.com/infrahq/infra/api"
-)
-
 type Settings struct {
 	Model
 	OrganizationMember
 
 	PrivateJWK EncryptedAtRest
```

Comment:
Doesn't have to be part of this PR, but maybe with should move these settings to the organization entity.

Reply:
This will be done in #4045